fcgiwrap fails to build

## Issue description
When building fcgiwrap derivation, the following error is raised at configure step:

    configure: error: systemd support requested but libraries not found

There was no recent change on fcgiwrap. Systemd is correctly listed as a dependency.
Useful link: https://github.com/gnosek/fcgiwrap/blob/master/configure.ac

### Steps to reproduce
Build the fcgiwrap derivation.


## Technical details

* System: NixOS: channel nixos-unstable
* Nix version: nix-env (Nix) 1.10
* Nixpkgs version: local clone of nixpkgs pointing to nixos-unstable

--- a/pkgs/servers/fcgiwrap/default.nix
+++ b/pkgs/servers/fcgiwrap/default.nix
@@ -13,6 +13,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ autoreconfHook systemd fcgi pkgconfig ];
 
+  # systemd 230 no longer has libsystemd-daemon as a separate entity from libsystemd
+  postPatch = ''
+    substituteInPlace configure.ac --replace libsystemd-daemon libsystemd
+  '';
+
   meta = with stdenv.lib; {
     homepage = https://nginx.localdomain.pl/wiki/FcgiWrap;
     description = "Simple server for running CGI applications over FastCGI";

